didcomm/protocols/forward/forward.py

```diff
@@ -71,13 +166,29 @@ async def unpack_forward(
 
     :return: Forward plaintext
     """
+    # TODO decrypt_by_all_keys ???
+    fwd_unpack_res = await unpack_anoncrypt(
+        json_str_to_dict(packed_msg), resolvers_config, False)
+
+    fwd_msg_dict = json_bytes_to_dict(fwd_unpack_res.msg)
+
+    if not is_forward(fwd_msg_dict):
+        raise MalformedMessageError(
+            MalformedMessageCode.INVALID_PLAINTEXT  # XXX or INVALID_MESSAGE
+        )
+
+    fwd_msg = ForwardMessage.from_dict(fwd_msg_dict)
+    msg = fwd_msg.attachments[0].data.json
```

Comment:
We need to check that the message contains non-empty list of JSON attachemnts

Reply:
that is covered by `is_forward` called few lines above (but there some TODOs inside that helper though)